Makefile

```diff
@@ -267,69 +257,53 @@ pylint:
 
-travis-push-cache-images:
-	${DOCKER} tag services_apihub:latest itisfoundation/apihub:cache
-	${DOCKER} tag services_webserver:latest itisfoundation/webserver:cache
-	${DOCKER} tag services_sidecar:latest itisfoundation/sidecar:cache
-	${DOCKER} tag services_director:latest itisfoundation/director:cache
-	${DOCKER} tag services_storage:latest itisfoundation/storage:cache
-	${DOCKER} tag services_webclient:build itisfoundation/webclient:cache
-	${DOCKER} tag services_webserver:latest itisfoundation/webserver:cache
-
-	${DOCKER} push itisfoundation/apihub:cache
-	${DOCKER} push itisfoundation/director:cache
-	${DOCKER} push itisfoundation/sidecar:cache
-	${DOCKER} push itisfoundation/storage:cache
-	${DOCKER} push itisfoundation/webclient:cache
-	${DOCKER} push itisfoundation/webserver:cache
-
-TRAVIS_PLATFORM_STAGE_VERSION=staging-$(shell date +"%Y-%m-%d").${TRAVIS_BUILD_NUMBER}.$(shell git rev-parse HEAD)
-TRAVIS_PLATFORM_STAGE_LATEST=staging-latest
-travis-push-staging-images:
-	${DOCKER} tag services_apihub:latest itisfoundation/apihub:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} tag services_webserver:latest itisfoundation/webserver:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} tag services_sidecar:latest itisfoundation/sidecar:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} tag services_director:latest itisfoundation/director:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} tag services_storage:latest itisfoundation/storage:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} tag services_apihub:latest itisfoundation/apihub:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} tag services_webserver:latest itisfoundation/webserver:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} tag services_sidecar:latest itisfoundation/sidecar:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} tag services_director:latest itisfoundation/director:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} tag services_storage:latest itisfoundation/storage:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} push itisfoundation/apihub:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} push itisfoundation/webserver:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} push itisfoundation/sidecar:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} push itisfoundation/director:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} push itisfoundation/storage:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} push itisfoundation/apihub:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} push itisfoundation/webserver:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} push itisfoundation/sidecar:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} push itisfoundation/director:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} push itisfoundation/storage:${TRAVIS_PLATFORM_STAGE_LATEST}
+.PHONY: push-cache	
+push-cache:
+	${DOCKER_COMPOSE} -f services/docker-compose.yml -f services/docker-compose.cache.yml push ${CACHED_SERVICES_LIST}
+
+
+# staging ----------------
+.PHONY: build-staging
+build-staging:
+	export DOCKER_IMAGE_PREFIX=itisfoundation/; \
```

Comment:
Why not to put these variables (DOCKER_IMAGE*) on the very top?? together with VCS_REF, BUILD_DATE, etc...